.github/workflows/release-artifacts.yaml

```diff
@@ -7,6 +7,11 @@ on:
       release_tag:
         description: "The release tag to target"
 
+env:
+  # We need to download the right LLVM tools on windows, and the version needs to match the one
+  # that rust uses. I don't know how to automate this...
+  LLVM_LINK: https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.8/clang+llvm-20.1.8-x86_64-pc-windows-msvc.tar.xz -o llvm.tar.xz
```

Comment:
It seems the version of LLVM is the same as it was before in the workflow, or am I missing something?